build_docker_images.sh

```diff
@@ -5,7 +5,48 @@ set -euo pipefail
 # Assumes that you have done a complete build so that all jars have been created, i.e.:
 #   ./mvnw clean install -P dse -DskipTests=true
 
-export SGTAG=v2.0.0-ALPHA1
+# Options:
+# -p - causes the images to be built for all supported platform architectures and pushed to
+#   Docker Hub (assumes you are logged in to Stargate Docker Hub account).
+#   This is intended to be used as part of automated builds.
+# -t <version> - overrides the default tag that will be applied to the image with the one
+#   you provide. By default the tag consists of the version is obtained from the parent
+#   pom.xml file, prepended with v, i.e. v2.0.0.
+
+#
+# Defaults
+#
+
+# generate for local platform, don't push
+DOCKER_FLAGS=
+
+# extract Stargate version from project pom file
+SGTAG="v$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)"
```

Comment:
This looks useful, something I could borrow :)

(Maven's version extractor is pretty good in general, although in this case it just prints the whole version so  bo parsing/extraction needed).

Reply:
Just to clarify, are you suggesting there is a better way to do this?

Reply:
Sorry, no, not at all. I was just remembering reading about logic Maven tooling extracts components of all kinds of exotic version Strings, and does it well & has documentation outlining how it works.
Which is really not relevant here, just a detour :)